component/main.jsonnet

```
@@ -202,7 +202,7 @@ local k8up_schedule =
 // Define outputs below
 {
   '00_namespace': namespace,
-  [if params.replicas >= 2 then '01_networkpolicy_infinispan']: networkpolicy_infinispan,
+  [if params.helm_values.networkPolicy.enabled && params.replicas >= 2 then '01_networkpolicy_infinispan']: networkpolicy_infinispan,
   [if params.ingress.enabled && params.helm_values.networkPolicy.enabled then '01_ingress_controller_ns_patch']: ns_patch,
```

To align this with the order of the previous line:
```suggestion
  [if params.helm_values.networkPolicy.enabled && params.ingress.enabled then '01_ingress_controller_ns_patch']: ns_patch,
```
or even combine both lines into one with an extra if (although this would be less readable IMO).

---

I will remove the ns patch code, in the next major release since not needed anymore: #188